### ISSUE TITLE
new update sporadically opening up the options page

### ISSUE THREAD

**Issue by anandijain** [wong2/chatgpt-google-extension#278]
i've noticed a few times that the options for the extension (chrome) will randomly open up, what's causing that?

**Comment by huyz** [comment]
Yeah, it seems the options page opens every time the browser (or profile) restarts for me.

**Comment by dannyhpy** [comment]
The Options page will open up every time your browser pulls an updated version of the extension.

**Comment by huyl** [comment]
Well that’s unnecessary. Should be fixed
